yt/data_objects/particle_trajectories.py

```diff
@@ -221,20 +229,20 @@ def _get_data(self, fields):
         fds = {}
         new_particle_fields = []
         for field in missing_fields:
-            fds[field] = dd_first._determine_fields(field)[0]
+            fds[field] = self.get_full_field_name(field)[0]
```

Comment:
This field isn't going to necessarily be a particle field, so I think using `_determine_fields` here is correct. I bet that will fix the test failure.

Reply:
I corrected it.